WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -9,9 +10,13 @@ def bad_api_parameter(e)
 
   DEFAULT_API_RESULT_LIMIT = 20
 
+  def current_resource_owner
```

Comment:
So it's basically `current_user`, but for API requests. 

Possible better names are `current_api_user`, `requesting_user` and probably others.